packages/paste-website/src/pages/components/badge/index.mdx

```diff
@@ -0,0 +1,465 @@
+    Error
+  </Badge>
+  <Badge as="a" href="#" variant="success">
+    Success
+  </Badge>
+  <Badge as="a" href="#" variant="new">
+    New
+  </Badge>
+</Box>
+`.trim()}
+</LivePreview>
+
+#### Badge with button functionality
+
+<LivePreview scope={{Box, Badge, InformationIcon, ErrorIcon, WarningIcon, SuccessIcon, NewIcon}} language="jsx">
+  {`<Box display="flex" columnGap="space40" rowGap="space60" flexWrap="wrap">
+  <Badge as="button" onClick={() => null} variant="default">
+    Default
+  </Badge>
+  <Badge as="button" onClick={() => null} variant="info">
+    Info
+  </Badge>
+  <Badge as="button" onClick={() => null} variant="warning">
+    Warning
+  </Badge>
+  <Badge as="button" onClick={() => null} variant="error">
+    Error
+  </Badge>
+  <Badge as="button" onClick={() => null} variant="success">
+    Success
+  </Badge>
+  <Badge as="button" onClick={() => null} variant="new">
+    New
+  </Badge>
+</Box>
+`.trim()}
+</LivePreview>
+
+## When to use a badge
+
+Use a badge when you want to visually highlight an attribute of an item.
+
+### Use cases
+
+#### Call attention to attributes of table items
+
+<LivePreview
+  scope={{
+    Box,
+    Table,
+    THead,
+    Td,
+    Th,
+    Tr,
+    Text,
+    TBody,
+    Badge,
+  }}
+  language="jsx"
+>
+  {tableExample.trim()}
+</LivePreview>
+
+#### Identify an item as as a beta release
+
+<LivePreview
+  scope={{
+    Box,
+    Badge,
+    Card,
+    NewIcon,
+    Heading,
+    Paragraph,
+  }}
+  language="jsx"
+>
+  {betaFeatureExample.trim()}
+</LivePreview>
+
+#### Highlight products
+
+<LivePreview
+  scope={{
+    Box,
+    Badge,
+    Grid,
+    Column,
+    RadioGroup,
+    Radio,
+    ProductElasticSIPTrunkingIcon,
+    ProductVoiceIcon,
+  }}
+  language="jsx"
+>
+  {settingsAndProducts.trim()}
+</LivePreview>
+
+### Do & Don't
+
+<DoDont>
+  <Do title="Do" body="Use 1-3 words as badge text.">
+    <Badge variant="new" as="span">
+      <NewIcon title="New badge component" size="sizeIcon10" />
+      New Component
+    </Badge>
+  </Do>
+  <Dont title="Don't" body="Don't use lengthy copy inside a badge.">
+    <Badge variant="error" as="span">
+      This is some very long text inside a badge that creates a very large badge.
+    </Badge>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Set the badge children to be inline.">
+    <Badge variant="new" as="span">
+      Inline badge
+    </Badge>
+  </Do>
+  <Dont title="Don't" body="Don't use multiline badge text.">
+    <Badge variant="new" as="span">
+      This
+      <Box as="br" /> is <Box as="br" />
+      not ideal
+    </Badge>
+  </Dont>
+</DoDont>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/badge - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {Badge} from '@twilio-paste/badge';
+
+const BadgeExample = () => (
+  <Badge variant="default" as="span">
+    Default Badge
+  </Badge>
+);
+```
+
+#### Props
+
+<Table>
```

Comment:
Doesn't seem to be anything too special about this props table, I'm curious why you didn't create a Markdown table? I think almost every props table uses markdown.